README.md

```diff
@@ -1 +1,7 @@
-# e-banking
\ No newline at end of file
+# Digital Bank 
+
+Para correr los test es necesario acceder al espacio de trabajo en Visual Studio Code, denominado E-BANKING WORKSPACE. 
```

Comment:
We defined the convention of **everything in english**, including documentation, class, functions and variable names, etc. Make sure everything is in english in this repo